src/modules/uuv_att_control/uuv_att_control.cpp

```diff
@@ -143,7 +160,9 @@ void UUVAttitudeControl::control_attitude_geo(const vehicle_attitude_s &attitude
 	float roll_u;
 	float pitch_u;
 	float yaw_u;
-	float thrust_u;
+	float thrust_x;
+	float thrust_y;
+	float thrust_z;
```

Comment:
Why not define this as `Vector3f` ?

Reply:
at a first glance I agreed with your suggestion but there seem to be several spots where we just use the first component (thrust_x) while the others remain unused, eg. the manual_control_setpoint are only 4D (x,y,z,r), hence y/z remain zero. 

so I would suggest to stay with this non perfect version here for the moment and change it when we improve the manual control part. Btw any suggestions on that? I think the regular remote control features only two 2D-sticks (x,y,z,r), thats why we usually use a gamepad.